std/uni.d

```diff
@@ -4332,6 +4332,7 @@ if (sumOfIntegerTuple!sizes == 21)
         }
     }
 
+    ///
     auto codepointTrie()(T[dchar] map, T defValue=T.init)
     {
         return buildTrie!(T, dchar, Prefix)(map, defValue);
```

Comment:
How about `/// ditto`-ing the `CodepointTrie` template below:

![image](https://user-images.githubusercontent.com/4370550/34903732-36702c4a-f838-11e7-91c8-5bde947a33f7.png)